tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/KafkaHelper.cs

```diff
@@ -212,8 +213,8 @@ internal static Scope CreateProducerScope(Tracer tracer, ITopicPartition topicPa
                         // TODO: we could pool these arrays to reduce allocations
                         // NOTE: the tags must be sorted in alphabetical order
                         var edgeTags = string.IsNullOrEmpty(topic)
-                                           ? new[] { $"group:{groupId}", "type:kafka", }
-                                           : new[] { $"group:{groupId}", $"topic:{topic}", "type:kafka", };
+                                           ? new[] { $"group:{groupId}", "type:kafka", "direction:in" }
+                                           : new[] { $"group:{groupId}", $"topic:{topic}", "type:kafka", "direction:in" };
```

Comment:
These must be sorted alphabetically
```suggestion
                                           ? new[] { "direction:in", $"group:{groupId}", "type:kafka" }
                                           : new[] { "direction:in", $"group:{groupId}", $"topic:{topic}", "type:kafka" };
```